src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-tabs/GeneViewTabs.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { RootState } from 'src/store';
+import { getEntityViewerActiveGeneTab } from 'src/content/app/entity-viewer/state/gene-view/entityViewerGeneViewSelectors';
+import { setActiveGeneTab } from 'src/content/app/entity-viewer/state/gene-view/entityViewerGeneViewActions';
+
+import Tabs, { Tab } from 'src/shared/components/tabs/Tabs';
+
+import styles from './GeneViewTabs.scss';
+
+const tabsData: Tab[] = [
+  { title: 'Transcripts' },
+  { title: 'Gene function' },
+  { title: 'Gene relationships' }
+];
+
+const DEFAULT_TAB = tabsData[0].title;
+
+type Props = {
+  selectedGeneTabName: string | null;
+  setActiveGeneTab: (selectedTabName: string) => void;
+};
+
+const GeneViewTabs = (props: Props) => {
+  const tabClassNames = {
+    default: styles.geneTab,
+    selected: styles.selectedGeneTab,
+    disabled: styles.disabledGeneTab,
+    tabsContainer: styles.geneViewTabs
+  };
+
+  const onTabChange = (selectedTabName: string) => {
+    if (selectedTabName === props.selectedGeneTabName) {
+      props.setActiveGeneTab(DEFAULT_TAB);
+      return;
```

Comment:
Won't it be clearer if we use an else statement rather than return from here?

Reply:
I always feel this approach is slightly cleaner that using else blocks. What's your opinion @azangru ?
Also, there is a discussion on stackexchange about this:
https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement

Reply:
There is value in an early return from an if statement when the body of the function is bulky and you feel that putting it all in an else block (and indenting it to the right) would decrease readability.

In this particular case though, I agree with Imran. Both branches of the if-condition are equally short; it is more readable to put the alternative branch inside the else block.

Reply:
Cool, I've updated it.